packages/strapi-helper-plugin/lib/src/components/ImgPreview/index.js

```diff
@@ -169,7 +169,7 @@ class ImgPreview extends React.Component {
 
     return (
       <div className="fileIcon" onDrop={this.handleDrop}>
-        <i className={`fa fa-file-${fileType}-o`} />
+        <i className={`far fa-file-${fileType}`} />
```

Comment:
Is it `far` or `fas`?

Reply:
This one is fas:
<img width="361" alt="Capture d’écran 2019-11-28 à 10 18 42" src="https://user-images.githubusercontent.com/3142866/69793338-858d9000-11c8-11ea-958a-5f0d33659aa8.png">

This one is far:
<img width="363" alt="Capture d’écran 2019-11-28 à 10 18 52" src="https://user-images.githubusercontent.com/3142866/69793339-858d9000-11c8-11ea-8862-d86d2d2f5deb.png">

So I'm pretty sure we want the regular/second one